models/Document.php

```diff
@@ -827,6 +827,7 @@ public function delete()
 
         //clear document from registry
         \Pimcore\Cache\Runtime::set(self::getCacheKey($this->getId()), null);
+        \Pimcore\Cache\Runtime::set('document_path_' . md5($this->getRealFullPath()), null);
```

Comment:
Should we create a `getPathCacheKey($path)` method?

Reply:
yes, please refactor it to `getPathCacheKey`

Reply:
done